zeroeq/http/server.h

```diff
@@ -167,13 +187,22 @@ class Server : public zeroeq::Receiver
      * @param func the callback function for serving the GET request
      * @return true if subscription was successful, false otherwise
      */
-    ZEROEQ_API bool register_( const std::string& event, const GETFunc& func );
+    ZEROEQ_API bool handleGET( const std::string& event, const GETFunc& func );
+
+    /**
+     * @overload
+     * @param event the event name to serve during receive()
+     * @param schema describes data layout of event
+     * @param func the callback function for serving the GET request
+     */
+    ZEROEQ_API bool handleGET( const std::string& event,
+                               const std::string& schema, const GETFunc& func );
```

Comment:
For empty events it would be more handy not having to pass an empty schema parameter.
Does it make sense to move scheme to the end of the parameter list with "" as a default value, or do you want to enforce an explicit schema for some reason?